src/server/routes/oauth.ts

```diff
@@ -800,6 +800,7 @@ router.get(
 			case 'callback':
 			// this is only used when we get back a standard auth `code` from the interaction code flow
 			// where we want to do the standard authentication callback
+			// falls through
```

Comment:
Informs eslint that the lack of a `break` for the `callback` case is intentional.